usr/lib/common/utility.c

```diff
@@ -292,9 +296,15 @@ CK_RV CreateXProcLock(char *tokname, STDLL_TokData_t *tokdata)
 
         /* create user lock file */
         if (strlen(tokname) > 0)
-            sprintf(lockfile, "%s/%s/LCK..%s", LOCKDIR_PATH, tokname, tokname);
+            snres = snprintf(lockfile, PATH_MAX, "%s/%s/LCK..%s",
+                             LOCKDIR_PATH, tokname, tokname);
         else
-            sprintf(lockfile, "%s/%s/LCK..%s", LOCKDIR_PATH, SUB_DIR, SUB_DIR);
+            snres = snprintf(lockfile, sizeof(lockfile), "%s/%s/LCK..%s",
+                             LOCKDIR_PATH, SUB_DIR, SUB_DIR);
+        if (snres < 0 || snres >= PATH_MAX) {
+            OCK_SYSLOG(LOG_ERR, "lock file path too long\n");
+            goto err;
```

Comment:
same here, if ock_syslog does not also emit an ock error log

Reply:
done